.github/workflows/test_docs_build.yml

```
@@ -0,0 +1,49 @@
+name: Documentation
+
+on:
+  workflow_dispatch:
+  pull_request:
+
+env:
+  PYBAMM_DISABLE_TELEMETRY: "true"
+  FORCE_COLOR: 3
+  PYBAMM_IDAKLU_EXPR_CASADI: OFF
+  PYBAMM_IDAKLU_EXPR_IREE: OFF
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  docs:
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Check out repository
+        uses: actions/checkout@v4
+
+      - name: Install Linux system dependencies
+        uses: awalsh128/cache-apt-pkgs-action@a6c3917cc929dd0345bfb2d3feaf9101823370ad # v1.4.2
+        with:
+          packages: graphviz pandoc
+          execute_install_scripts: true
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.11"
+          cache: 'pip'
+
+      - name: Install PyBaMM with [docs,dev,all]
+        run: |
+          python -m pip install --upgrade pip
+          pip install -e .[docs,dev,all]
+
+      - name: Run doctests
+        run: python -m nox -s doctests
```

```suggestion
      - name: Run doctests
        run: python -m pytest --doctest-plus src
```

I think this was missed from https://github.com/pybamm-team/PyBaMM/pull/4686#discussion_r1890504705.